validate_key.py

```diff
@@ -21,9 +20,13 @@ def main(args):
         return {'valid': False}
     [key_created, age, source, campaign, hash] = args.KEY.split('.')
     m = hashlib.sha256()
+
+    script_settings = get_secret('ak-partner-rsvp')
+    secret = script_settings['SECRET']
```

Comment:
Can we name this something a little more descriptive?

Reply:
changed to `key_hash_secret`